view/frontend/requirejs-config.js

```diff
@@ -11,5 +11,10 @@ var config = {
                 'Magento_CheckoutAgreements/js/model/place-order-mixin': true
             }
         }
+    },
+    map: {
+        '*': {
+            'adyenCheckout':  'https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.4.0/adyen.js'
```

Comment:
does it matter that we always use the `-live` endpoint, instead of using the proper one? (`-test` or `-live`)